app/src/main/java/com/yalta/activities/fragments/DriverRoutesFragment.kt

```diff
@@ -0,0 +1,77 @@
+package com.yalta.activities.fragments
+
+import android.graphics.drawable.ClipDrawable
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.DividerItemDecoration
+import com.google.android.material.datepicker.MaterialDatePicker
+import com.yalta.R
+import com.yalta.databinding.FragmentDriverRoutesBinding
+import com.yalta.di.YaltaApplication
+import com.yalta.viewmodel.DriverRoutesViewModel
+import kotlinx.android.synthetic.main.fragment_driver_routes.*
+import org.joda.time.DateTime
+import org.joda.time.DateTimeZone
+import javax.inject.Inject
+
+class DriverRoutesFragment : Fragment() {
+    @Inject
+    lateinit var viewModel: DriverRoutesViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val binding: FragmentDriverRoutesBinding =
+            DataBindingUtil.inflate(inflater, R.layout.fragment_driver_routes, container, false)
+
+        YaltaApplication.appComponent.inject(this)
+
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this
+
+        binding.routeRecyclerView.addItemDecoration(DividerItemDecoration(context, ClipDrawable.HORIZONTAL))
```

Comment:
Because I hate creating custom styles and that's the only way to move it to xml - standard `RecyclerView` does not have built-in xml opportunity to add dividers

https://medium.com/sodalabs/recyclerview-pro-tips-add-line-divider-in-xml-cd5c2a4df110